projects/core/src/features-config/utils/feature-config-utils.ts

```diff
@@ -30,6 +35,8 @@ export function isFeatureEnabled(config: unknown, feature: string): boolean {
     const featureConfig = config.features[feature];
```

Comment:
`fatrure` here can now contain '!' at the beginning, so you should take it into an account